Specs/Scene/ModelExperimental/ModelExperimentalUtilitySpec.js

```diff
@@ -32,6 +33,64 @@ describe("Scene/ModelExperimentalUtility", function () {
     expect(Matrix4.equals(computedTransform, Matrix4.IDENTITY)).toEqual(true);
   });
 
+  it("createBoundingSphere works", function () {
+    var mockPrimitive = {
+      attributes: [
+        {
+          semantic: "POSITION",
+          max: new Cartesian3(0.5, 0.5, 0.5),
+          min: new Cartesian3(-0.5, -0.5, -0.5),
+        },
+      ],
+    };
+    var translation = new Cartesian3(50, 50, 50);
+
+    var modelMatrix = Matrix4.fromTranslationRotationScale(
+      new TranslationRotationScale(
+        translation,
+        Quaternion.IDENTITY,
+        new Cartesian3(1, 1, 1)
+      )
+    );
+    var boundingSphere = ModelExperimentalUtility.createBoundingSphere(
+      mockPrimitive,
+      modelMatrix
+    );
+
+    expect(boundingSphere.center).toEqual(translation);
+    expect(boundingSphere.radius).toEqual(0.8660254037844386);
```

Comment:
Maybe make these `.toEqualEpsilon()` checks just in case different browsers implement `sqrt()` differently.